tests/integration/components/permission-language-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('permission-language', 'Integration | Component | permission language', {
+  integration: true
+});
+
+test('it renders', function(assert) {
```

Comment:
I just added tests to ensure the component is returning the proper language for each provider.